posthog/queries/sessions.py

```diff
@@ -195,3 +197,16 @@ def _session_dist(self, base_query: Query, params: QueryParams) -> List[Dict[str
             for index in range(len(DIST_LABELS))
         ]
         return result
+
+
+def scale_time_series(data: List[float]) -> Tuple[List, str]:
+    avg = sum(data) / len([value for value in data if value > 0])
+    minutes, _ = divmod(avg, 60.0)
+    hours, _ = divmod(minutes, 60.0)
```

Comment:
Since mod is discarded, you can just use do integer division with `//`.

Reply:
yup!